docs/src/pages/getting-started/supported-platforms/supported-platforms-de.md

```diff
@@ -14,7 +14,7 @@ Because Googlebot uses a web rendering service (WRS) to index the page content,
 
 ## Server
 
-Because Material-UI supports server-side rendering, we need to support the latest, stable releases of [Node.js](https://github.com/nodejs/node). We try to support the [last active LTS version](https://github.com/nodejs/Release#lts-schedule1). Right now, we support **node v6.x** and newer versions.
+Because Material-UI supports server-side rendering, we need to support the latest, stable releases of [Node.js](https://github.com/nodejs/node). We try to support the [last active LTS version](https://github.com/nodejs/Release#lts-schedule1). Right now, we support **node v8.x** and newer versions.
```

Comment:
Question about that statement: Shouldn't it always have been "oldest LTS version"? The "latest active LTS version" would be node 10 now.